solutions/cses-2428.mdx

```diff
@@ -0,0 +1,70 @@
+---
+id: cses-2428
+source: CSES
+title: Subarray Distinct Values
+author: Sofia Yang
+---
+
+**Time Complexity:** $\mathcal{O}(N)$
+
+
+<LanguageSection>
+
+<JavaSection>
+
+
+``` java
+import java.io.*;
+import java.util.*;
+CodeSnip{Kattio}
+public class cses2428 {
+	public static void main(String[] args) throws IOException {
+		Kattio io = new Kattio();
+		int N = io.nextInt();
+		int K = io.nextInt();
+		int[] A = new int[N];
```

Comment:
Can you use variable names that are more than one letter long (for variables A, l, r)?